app/src/main/java/de/danoeh/antennapod/fragment/preferences/MainPreferencesFragment.java

```diff
@@ -145,5 +145,8 @@ private void setupSearch() {
                 .addBreadcrumb(PreferenceActivity.getTitleOfPage(R.xml.preferences_gpodder));
         config.index(R.xml.preferences_notifications)
                 .addBreadcrumb(PreferenceActivity.getTitleOfPage(R.xml.preferences_notifications));
+        config.index(R.xml.feed_settings)
+                .addBreadcrumb(PreferenceActivity.getTitleOfPage(R.xml.feed_settings));
```

Comment:
If you just use getString here, you don't need to add the title to the activity, I think.

Reply:
I could but I felt it would be best for consistency.